File: codechef/jan14/PLZLYKME.cpp
#include<cstdio>
#include<iostream>
using namespace std;
int main()
{
    int t;
    long long int l,d,s,c,a;
    scanf("%d",&t);
    while(t--)
    {
        cin>>l>>d>>s>>c;
        a=s;
        c=1+c;
        d=d-1;
        while(d)
        {
            if(d%2==1)
                a=a*c;
            c=c*c;
            d=d/2;
            if(a>=l)
                break;
        }
        if(a>=l)
            cout<<"ALIVE AND KICKING"<<endl;
        else
            cout<<"DEAD AND ROTTING"<<endl;
    }
    return 0;
}

File: codechef/jan14/PLZLYKME.py
t=input()
while(t>0):
    a=raw_input()
    b=[]
    b=a.split(" ")
    l=int(b[0])
    d=int(b[1])
    s=int(b[2])
    c=int(b[3])
    c=1+c
    d=d-1
    a=1
    t=t-1
    while(d>0):
        if(d%2==1):
            a=a*c
        c=c*c
        d=d/2
        if(a>=l):
            break
    a=a*s
    if(a>=l):
        print "ALIVE AND KICKING"
    else:
        print "DEAD AND ROTTING"
